public/locales/mxb/translation.json

```diff
@@ -94,33 +94,33 @@
+    "10answer": "Saa. Ja sa’a chití ní tu’un nuu tutu ya’a nduu kuni ka’an ja ñama kuni ní nuu tutu ní. Enlace ja tetiñu nuu iyo tutu ní jiniñu’un in PIN ja ni kaji ní de vachi ni tetiñu nuu yokaa ní axi nuu in correo electrónico ja yita’an ji tutu vacuna ní. <br />Código QR kuu in SMART Health Card (Tarjeta salud ja ndichí), kuu in copia tutuní ja ñu’un va’a. Iyo ka tu’un nuu <1>https://smarthealth.cards</1>. Tágua koto tu’un siki ní, código QR vachi ni kuu katiñu jín in kaa ndichí ja satiñu jín SMART Health Card. ",
     "11question": "A kuu nasa’a jee sa PIN sa",
-    "11answer": "Saa. Iyo 24 hora nde kivi kenda enlace nuu ni tágua chi’in ní PIN ní ja kuu kumi número de sukua kivi ní nuu Tutu nuu kaa ndichí siki tu’un nasa iyo ní jín kue’e COVID-19 (vachi nuu yu’u Inglés). Tú ma sa’a ní, enlace ja kivi ní nuu iyo tutu ní naa, vese saa kuu ndije’e tuku ní de nasa jee ní PIN ní de kuatiñu ní Tutu nuu kaa ndichí siki tu’un nasa iyo ní jín kue’e COVID-19 (vachi nuu yu’u Inglés).<br />Tú nduu na’an ní PIN ní, nuu 24 kuu ndije’e tuku ní de sa’a n+i in PIN jee.<br /><5>NOTA: Tú ja ni kivi ní de ni nani’in ní tutu ní de ni tava’a ní código QR ní, nduka naa ji.</5>",
+    "11answer": "Saa. Iyo 24 hora nde kivi kenda enlace nuu ni tágua chi’in ní PIN ní ja kuu kumi número de sukua kivi ní nuu <1>Tutu nuu kaa ndichí siki tu’un nasa iyo ní jín kue’e COVID-19</1> (vachi nuu yu’u Inglés). Tú ma sa’a ní, enlace ja kivi ní nuu iyo tutu ní naa, vese saa kuu ndije’e tuku ní de nasa jee ní PIN ní de kuatiñu ní <3>Tutu nuu kaa ndichí siki tu’un nasa iyo ní jín kue’e COVID-19</3> (vachi nuu yu’u Inglés). <br />Tú nduu na’an ní PIN ní, nuu 24 kuu ndije’e tuku ní de sa’a n+i in PIN jee. <br /><5>NOTA: Tú ja ni kivi ní de ni nani’in ní tutu ní de ni tava’a ní código QR ní, nduka naa ji.</5>",
     "12question": "A stuu tutu ya’a siki nasa iyo nda vacuna sa",
-    "12answer": "Saa. Ku in nuu síin síin kaa () tágua chu’un ní your Tutu nuu kaa ndichí siki tu’un nasa iyo ní jín kue’e COVID-19 () de kuu tava ní axi ste’en ní nuu in kaa ndichí. Suni kuu ste’en ní tarjeta COVID-19 ja ni taji Centers for Disease Control and Prevention (CDC, Ve’e nuu jito ji jekani nda kue’e) nuu ní axi tutu siki tatan ní ja iyo nuu ñuu.<br />Nduu jiniñu’un ja kuneva’a ní in Tutu nuu kaa ndichí siki tu’un nasa iyo ní jín kue’e COVID-19 (). Kuu in ja kuu ni’in ní tú kúni ní tu’un siki nasa iyo ni nuu COVID-19 de kuu in tutu ja iyo nuu kaa ndichí. Kuu in ja kuu kuetiñu ní tágua ste’en ní ja ni kani vacuna ní. Ñuu ma tava ji in pasaporte ja ndíso tu’un siki ja ya’a.",
```

Comment:
I'm noticing multiple instances of () in this translation document. Should those be removed? or were those part of the translation document returned by NWIS?